handler/webhook.go

```diff
@@ -0,0 +1,24 @@
+package handler
+
+import (
+	"log"
+	"net/http"
+
+	"github.com/forjadev/gun-organization/service"
+	"github.com/gin-gonic/gin"
+)
+
+func WebhookReceiverHandle(c *gin.Context) {
```

Comment:
I know that we discussed on meetings about this being a webhook receiver, but we added this name as a placeholder while we do not think on a better name, 
We should consider naming the function more descriptively. For example, instead of `WebhookReceiverHandle`, you could name it `HandleGitHubWebhook`, what do you think? I think it describes better what it does and handle!